collections/alos-2-palsar-2-stripmap-for-economy.yaml

```diff
@@ -19,11 +19,11 @@ Resolution: |
  SM1: 3m
 GeographicalCoverage: |
   Paris, France SM3: [49.004, 3.111 : 48.519, 3.672] (Upper Left: Bottom Right )  
-  Dunkirk, France SM3: [51.462, 1.775 : 54.447,3.186]   
-  LA SM1:  (34.226, -118.002 : 33.707, -117.869)  
-  Beijing, China SM1, SM3:  [40.129, 116.283 : 39.637, 117.247]  
-  Haneda SM1: (35.965, 139.651: 35.267, 139.897)
-TemporalAvailability: 2019-03-01  -  2020-12-31
+  Dunkirk, France SM3: [51.587, 1.127 : 50.466, 3.204]   
```

Comment:
Where did this info come from?

Reply:
Dorothy checked the data. The corrected bbox is the one from the data. The original one in the file had a mistake and was not valid.